### PR TITLE
RNN, Final task

### PR DESCRIPTION
**Pull request by chinarev**
1) Подбор гиперпараметров: Best validation accuracy: 0.88, best hyperparams: Hyperparams(learning_rate=0.01, anneal_epochs=1, reg=1e-05)
2) Cвободное упражнение Val accuracy: 0.896594
3) Final test accuracy -  0.8993546404425322

### CODE REVIEW

**Comment by chinarev** [comment]
RNN final task done
LSTMTagger -> Test accuracy is 95.7%
Bidirectional LSTM -> Test accuracy for biderectional LSTM is 95.4%
Final model -> Test accuracy for biderectional LSTM with pretrained embeddings is 96.9%

**Comment by chinarev** [comment]
Перенес RNN в свою папку
